Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -85,57 +85,65 @@ - (void)layoutSubviews {
 
     self.shapeLayer.frame = CGRectMake(paddingX, 0, BPKCalendarCellSpacing.cellCircleHeight, BPKCalendarCellSpacing.cellCircleHeight);
 
+    CGFloat underflow = 0;
+    CGFloat overflow = 0;
+    CGFloat overflowLength = [[UIScreen mainScreen] bounds].size.width;
     switch (self.rowType) {
-    case RowTypeStart:
-        if (!isRTL) {
-            corners = UIRectCornerTopLeft | UIRectCornerBottomLeft;
-        } else {
-            corners = UIRectCornerTopRight | UIRectCornerBottomRight;
-        }
-        cornerRadii = CGSizeMake(height / 2.0, height / 2.0);
-        break;
-
-    case RowTypeEnd:
-        if (!isRTL) {
-            corners = UIRectCornerTopRight | UIRectCornerBottomRight;
-        } else {
-            corners = UIRectCornerTopLeft | UIRectCornerBottomLeft;
-        }
-        cornerRadii = CGSizeMake(height / 2.0, height / 2.0);
-        break;
-
-    case RowTypeBoth:
-        corners = UIRectCornerAllCorners;
-        cornerRadii = CGSizeMake(height / 2.0, height / 2.0);
+        case RowTypeStart:
+            if (self.selectionType != SelectionTypeLeadingBorder) {
+                underflow = overflowLength;
```

Comment:
Isn't this too much? i.e. it should be something like ~16-40pts but this will be more like ~350pt?

Reply:
The maximum it could need to be would be `6*cellWidth + bpkSpacingBase`

Reply:
Right, my thoughts on this are in the above conversation